docs/_config.yml

```diff
@@ -0,0 +1,6 @@
+title: kube-hunter
+description: Kube-hunter hunts for security weaknesses in Kubernetes clusters
+logo: https://raw.githubusercontent.com/aquasecurity/kube-hunter/master/kube-hunter.png
+show_downloads: false
+#google_analytics: 
```

Comment:
You could ask Shirley for a google analytics ID

Reply:
done